packages/rbflabs-agoric-react-components/src/contexts/wallet/hooks/use-wallet-notifications/utils.ts

```diff
@@ -0,0 +1,132 @@
+import { WalletNotification, WalletOfferStatus, WalletState } from '../../types';
+
+export function getConnectNotification(
```

Comment:
What about having one exported item per file where each file is named after this item?